languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -37493,6 +37493,57 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+      <rule id="A_OU_EN_MOYEN_DE_TRANSPORT" name="a ou en moyen de transport" default="temp_off">
+          <antipattern>
+              <token>en</token>
+              <token regexp="yes">vélo|byciclette|moto|cheval|skis</token>
+              <token postag="Y"/>
+          </antipattern>
+          <antipattern>
+              <token>en</token>
+              <token regexp="yes">vélo|byciclette|moto|cheval|skis</token>
+              <token regexp="yes">(?-i)[A-Z].*</token>
+          </antipattern>
+          <antipattern>
+              <token>en</token>
+              <token regexp="yes">vélo|byciclette|moto|cheval|skis</token>
+              <token postag="J.*" postag_regexp="yes"/>
+          </antipattern>
+          <antipattern>
+              <token postag="V.*" postag_regexp="yes" regexp="yes" inflected="yes" skip="-1">transformer|déguiser|métamorphoser|changer</token>
+              <token>en</token>
+              <token regexp="yes">vélo|byciclette|moto|cheval|skis</token>
+          </antipattern>
+          <antipattern>
+              <token postag="V.*" postag_regexp="yes" regexp="yes" inflected="yes">transformer|déguiser|métamorphoser|changer</token>
+              <token>en</token>
+              <token regexp="yes">vélo|byciclette|moto|cheval|skis</token>
+          </antipattern>
+          <antipattern>
+              <token>en</token>
+              <token regexp="yes">vélo|byciclette|moto|cheval|skis</token>
+              <token regexp="yes">de|d'|cross|neige|trial|freeride|freestyle|rando|racing</token>
+          </antipattern>
+          <antipattern>
+              <token>en</token>
+              <token regexp="yes">vélo|byciclette|moto|cheval|skis</token>
+              <token regexp="yes">tout|à</token>
+              <token regexp="yes">chemin|terrain|cailloux</token>
+          </antipattern>
+          <pattern>
+              <marker>
+                  <token>en</token>
+              </marker>
+              <token regexp="yes">vélo|byciclette|moto|cheval|skis</token>
```

Comment:
The rule editor results gave me some ideas: Does this rule work also for `snowboard|skateboard|VTT|raquettes|crampons` or some of those?

Do the many rule editor results all look okay to you?

Reply:
It would have been great if it worked for these cases too as well as "tandem, tricycle, surf..." but for some reason, it doesn't work. 
The last time I checked, all the results were okay.